src/App.css

```diff
@@ -53,3 +54,78 @@
     max-width: 100%;
   }
 }
+.imgArt {
+  max-width: 70%;
+  height: auto;
+  padding: 3px;
+  box-shadow: 0px 0px 15px 0px grey;
+  margin-left: 1%;
+}
+.articleDiv {
+  display: flex;
+}
+
+h1 {
+  color: #111;
+  font-family: "Helvetica Neue", sans-serif;
+  font-size: 26pt;
+  font-weight: bold;
+  letter-spacing: 0px;
+  line-height: 1;
+  text-align: left;
+}
+h2 {
+  color: #d54d7b;
+  font-family: "Great Vibes", cursive;
+  font-size: 28pt;
+  line-height: 1;
+  font-weight: normal;
+  margin-bottom: 3px;
+  margin-top: 4px;
+  text-align: left;
+  text-shadow: 0 1px 1px #fff;
+  margin-left: 0.75%;
+}
+
+.pSpec {
+  color: #685206;
+  font-family: "Helvetica Neue", sans-serif;
+  font-size: 14px;
+  line-height: 24px;
+  margin: 0 1% 24px;
+  max-width: 85%;
+  text-align: justify;
+  text-justify: inter-word;
+}
+
+p {
+  color: #111;
+  font-family: "Open Sans", sans-serif;
+  font-size: 16px;
+  line-height: 28px;
+  margin: 0 1% 8px;
+  text-align: justify;
+  max-width: 97%;
+}
+
+.comDiv {
+  box-shadow: 0px 0px 2px 0px grey;
+  margin: 0 1% 2px;
+}
+
+.spanVotes {
+  float: right;
+  font-size: 0.8rem;
+  text-align: right;
+}
+.frontPgTitle {
+  text-decoration: none;
+  text-align: left;
+  font-family: "Great Vibes", cursive;
+  font-size: 0.9em;
+  color: #111;
+}
+
+.imgAvatarPg {
+  max-width: 15%;
+}
```

Comment:
The design elements here are really excellent and responsive - really good work.